enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -395,6 +396,24 @@ const theme = {
   },
 
   components: {
+    // Dashboard
+    dashboard: {
```

Comment:
I don't see `dashboard.backgroundColor` here. Should we add it? 

https://github.com/metabase/metabase/blob/a8d5ea6437db8eafd8661a6d2a2a65664b892a60/enterprise/frontend/src/embedding-sdk/types/theme/index.ts#L83

Reply:
I indeed forgot that, thanks!